puppet/modules/slave/manifests/init.pp

```diff
@@ -393,6 +393,13 @@
     group  => 'jenkins',
     source => 'puppet:///modules/slave/katello-ca.cert',
   }
+  file { '/home/jenkins/.config/copr':
+    ensure   => file,
+    mode     => '0644',
```

Comment:
I think 640 might be better since it has tokens

Reply:
+1